computestorage/helpers.go

```diff
@@ -0,0 +1,151 @@
+package computestorage
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/Microsoft/go-winio/pkg/security"
+	"github.com/Microsoft/go-winio/vhd"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/windows"
+)
+
+// SetupContainerBaseLayer is a helper to setup a containers scratch. It
+// will create and format the vhdx inside and the size is configurable with the sizeInGB
+// parameter.
+func SetupContainerBaseLayer(ctx context.Context, layerPath, vhdName string, sizeInGB uint64) error {
+	var (
+		baseVhdPath = filepath.Join(layerPath, vhdName+"-base.vhdx")
+		diffVhdPath = filepath.Join(layerPath, vhdName+".vhdx")
+		hivesPath   = filepath.Join(layerPath, "Hives")
+		layoutPath  = filepath.Join(layerPath, "Layout")
+	)
+
+	if _, err := os.Stat(hivesPath); err == nil {
+		os.RemoveAll(hivesPath)
+	}
+	if _, err := os.Stat(layoutPath); err == nil {
+		os.RemoveAll(layoutPath)
+	}
```

Comment:
Why is this needed?

Reply:
If the files exist already, the compute storage calls fail

Reply:
Are they created as part of those calls? If so, which one?

Reply:
I'll add some comments detailing this